tests/ci_build/ci_build.sh

```diff
@@ -176,7 +180,7 @@ ${DOCKER_BINARY} run --rm --pid=host \
     -e "CI_BUILD_UID=$(id -u)" \
     -e "CI_BUILD_GROUP=$(id -g -n)" \
     -e "CI_BUILD_GID=$(id -g)" \
-    -e "CUDA_ARCH=-gencode arch=compute_52,code=[sm_52,compute_52] --fatbin-options -compress-all" \
+    -e "CUDA_ARCH=-gencode arch=compute_52,code=[sm_52,compute_52] -gencode arch=compute_61,code=sm_61 --fatbin-options -compress-all" \
```

Comment:
6.1 is Pascal based GPUs and thus would have no effect on our CI. Could you elaborate?

Reply:
This will produce backwards compatible binaries, including the dp4a instruction, that will work on p3 instances.